### PR TITLE
chore: rename thaum to tari

### PR DESCRIPTION
**Pull request by CjS77**
Description
---

The whimsical and Discworld-related placeholder name for DAN tokens was `Thaum`. This has since been renamed to Tari. (And L1 tokens are now Minotari).

### CODE REVIEW

**Comment by CjS77** [comment]
ACK

File: src/RFC-0310_SubmarineSwaps.md [hunk 1]
```diff
@@ -290,56 +295,56 @@ This phase allows Alice and Bob to commit to using their keys.
...
 
-### Thaum payment
+### Tari payment
 
-When Bob sees that the Tari UTXO that Alice created is mined on the Tari blockchain with the correct script, Bob can
-publish the Thaum transaction containing the Thaum commitment with the aggregate key \\(X = X_a + X_b + k_i \cdot G \\).
+When Bob sees that the Minotari UTXO that Alice created is mined on the Minotari blockchain with the correct script, Bob can
+publish the Tari transaction containing the Tari commitment with the aggregate key \\(X = X_a + X_b + k_i \cdot G \\).
 
 ### Claim Tari
```
Comment (SWvheerden) [inline src/RFC-0310_SubmarineSwaps.md:320]:
```suggestion
### Claim Minotari 
```

File: src/RFC-0310_SubmarineSwaps.md [hunk 2]
```diff
@@ -290,56 +295,56 @@ This phase allows Alice and Bob to commit to using their keys.
...
-missing key \\(x_b \\) to claim the Thaum yet, but it will be revealed when Bob claims the Tari. 
+When Alice sees that the Tari commitment that Bob created is confirmed on the second layer containing the correct aggregate 
+key \\(X\\), she can provide Bob with the required `pre_image` to spend the Minotari UTXO. She does not have the 
+missing key \\(x_b \\) to claim the Tari yet, but it will be revealed when Bob claims the Minotari. 
 
-Bob can now supply the `pre_image` and his Monero private key as transaction input to unlock the script.
+Bob can now supply the `pre_image` and his Tari private key as transaction input to unlock the script.
 
-### Claim Thaum
+### Claim Minotari
```
Comment (SWvheerden) [inline src/RFC-0310_SubmarineSwaps.md:328]:
```suggestion
### Claim Tari
```

**Review by SWvheerden — COMMENTED** [review]
Looking good, just two missed renames

**Comment by SWvheerden** [comment]
We should probably expand this to the other rfc's.
A lot of the Minotari rfc's still say Tari.

**Comment by CjS77** [comment]
> We should probably expand this to the other rfc's. A lot of the Minotari rfc's still say Tari.

Yeah, but not this PR. This one is to remove the deprecated Thaum moniker.

**Comment by SWvheerden** [comment]
ACK

**Review by SWvheerden — APPROVED** [review]
ACK

**Pull request merged**